packages/core/content-type-builder/admin/src/utils/axiosInstance.js

```diff
@@ -34,4 +35,20 @@ instance.interceptors.response.use(
   }
 );
 
-export default instance;
+const wrapper = {};
```

Comment:
You could put that in the helper plugin only once and export it to use instead of copy/pasting it probably

Reply:
ok done, thanks